src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/ComInterfaceMarshaller.cs

```diff
@@ -35,6 +40,11 @@ public static unsafe class ComInterfaceMarshaller<T>
             return CastIUnknownToInterfaceType(unknown);
         }
 
+        /// <summary>
+        /// Convert a COM interface pointer to a managed object.
+        /// </summary>
+        /// <param name="unmanaged">The COM interface pointer</param>
+        /// <returns>A managed object that represents the passed in COM interface pointer, or the managed object represented by the passed in pointer.</returns>
```

Comment:
I'm guessing the second part is referring to if it's unwrapping a pointer to a managed object from .Net? If so I think that could be clearer.